templates/helm/templates/_namespaced-install.tpl

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Please rename this file to `.yaml`. `.tpl` is reserved for Go template files.

Also perhaps since this file is used to patch, we could instead call it `controller-role-kind-patch.yaml`?

Reply:
Sure